configuration/blockly/rules-blockly-before-using.md

```diff
@@ -145,14 +156,6 @@ After pressing _Enter_ all found blocks that contain the search term are marked
 
 ![blockly-search](../images/blockly/blockly-search.png)
 
-### Zoom the workspace
```

Comment:
I am happy with this PR except that I only notice now that you have seemed to remove the zooming controls or am I wrong in the code PR which I seemed to have overlooked by then?

<img width="56" alt="image" src="https://github.com/openhab/openhab-docs/assets/5937600/114701a1-2247-4afa-b46c-3b72b76ac6f1">

I always liked those and use them quite often.

Reply:
The zoom controls are still there in the software. In the docs it was moved to the top. Do a search for "zoom".

<img width="1244" alt="image" src="https://github.com/openhab/openhab-docs/assets/2554958/69296269-bf8f-45fb-984a-bb2a3b7b99fd">

Reply:
<img width="958" alt="image" src="https://github.com/openhab/openhab-docs/assets/2554958/be371dcb-1450-4dc7-aaf1-57fe4e39c16f">